### PR TITLE
Fixes some broken URLs and unnecessary "deletes" 

### PR DESCRIPTION
**Pull request by jasvir**

### CODE REVIEW

File: src/pages/address/index.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
 </div>
 <div class="test">
   <strong>Location</strong>
-  <p>Sources from the raw location and sinsk into various javascript functions.</p>
+  <p>Sources from the raw location and sinkis into various javascript functions.</p>
```
Comment (paradoxengine) [inline src/pages/address/index.html]: Since you are fixing this, sinks

File: src/tests/dom/data/sources/localStorage/array.tmpl [hunk 2]
```diff
@@ -5,6 +5,5 @@ setTimeout(function() {
   trigger(localStorage['badValue']); // Using the async trigger.
 
   // Clean up the localStorage to avoid causing side-effect on subsequent tests.
-  delete window.localStorage;
```
Comment (paradoxengine) [inline src/tests/dom/data/sources/localStorage/array.tmpl]: Why kill this? (might make sense, but worth checking)

**Comment by paradoxengine** [comment]
Minor nit, otherwise CL looks good, can merge once fixed.

File: src/pages/address/index.html (hunk 1 above)
Comment (jasvir) [inline src/pages/address/index.html]: Done

**Comment by jasvir** [comment]
Note as mentioned in the bug description, the removal of the "delete window.localStorage" is deliberate and is one of the things being fixed.

See http://jsfiddle.net/2d000fjt/ to repro the bug - it may only happen to be working for the web security scanner if localStorage was being frozen or mocked in some way that delete window.localStorage didn't end up deleting the localStorage object entirely rather than just emptying it.

**Comment by paradoxengine** [comment]
Thank you Jas!

**Pull request merged**